pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2021 Markus KARG. All rights reserved.
+    Copyright (c) 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
I veto the replacement of my copyright, just as Oracle would veto theirs in such a case.

Reply:
I just pulled that from https://github.com/jakartaee/rest/commit/e5f23e8cb328aee010f9ea1dae85ec8186d7cf93#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L4 which is in release-4.0 as well. That said, I'm happy to change it back. I don't really know what it should be so keeping it as is, e.g. "Copyright (c) 2021 Markus KARG. All rights reserved." seems fine to me.

Reply:
Thank you for keeping it. Once fixed I am +1 with the rest of this PR.